### PR TITLE
geni 1.0.6 (new formula)

### PR DESCRIPTION
**Pull request by emilpriver**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/g/geni.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    ENV["DATABASE_URL"] = "sqlite3://test.sqlite"
+    system bin/"geni", "create"
+    assert_predicate testpath/"test.sqlite", :exist?, "failed to create test.sqlite"
```
Comment (chenrui333) [inline Formula/g/geni.rb:17]: can we run a version assert as well? the rest lgtm

Comment (emilpriver) [reply Formula/g/geni.rb:17]: Hey, ofc! on it :D

Comment (emilpriver) [reply Formula/g/geni.rb:17]: Should be added now. I force pushed a new commit for the commit history :D

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@emilpriver thanks!

**Review by branchv — APPROVED** [review]

**Comment by emilpriver** [comment]
Hey, Big thanks for the merge!

**Pull request merged**